packages/frontend-shared/src/components/Card.vue

```diff
@@ -8,7 +8,7 @@
       'bg-gray-50 border-gray-100 pointer-events-none': disabled
     }"
     data-cy="card"
-    :tabindex="disabled ? -1 : 1"
+    :tabindex="disabled ? -1 : 0"
```

Comment:
The tabindex of 1 pulled the cards out of the natural tab order. Talked with @marktnoonan, the cards are going to get a larger markup change in the future.